tensorflow/lite/micro/kernels/fully_connected.h

```diff
@@ -65,14 +65,14 @@ TfLiteStatus CalculateOpDataFullyConnected(
 // (reference or optimized) must define this function.
 TfLiteRegistration Register_FULLY_CONNECTED();
 
-#if defined(CMSIS_NN)
+#if defined(CMSIS_NN) || defined(HEXAGON)
 // The Arduino is a special case where we use the CMSIS kernels, but because of
 // the current approach to building for Arduino, we do not support -DCMSIS_NN as
 // part of the build. As a result, we use defined(ARDUINO) as proxy for the
 // CMSIS kernels for this one special case.
```

Comment:
unrelated to this change, but can you delete this comment. It is no longer valid since the #defined(ARDUINO) is now handled in the Arduino examples repo.